common/app/routes/challenges/Test-Suite.jsx

```diff
@@ -8,6 +8,24 @@ const propTypes = {
   tests: PropTypes.arrayOf(PropTypes.object)
 };
 
+function getAccessibleText(err, pass, text) {
+  let accessibleText = '';
+
+  // Determine test status (i.e. icon)
+  if (err) {
+    accessibleText += 'Error ';
+  } else if (pass) {
+    accessibleText += 'Pass';
+  } else {
+    accessibleText += 'Waiting';
```

Comment:
This logic can be simplified. If Waiting is the default then this last conditional can be removed and the `accessibleText` can start out as `Waiting`.

Then the two conditionals above just overwrite the string.